src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/machines/enchanting/AutoDisenchanter.java

```diff
@@ -50,13 +51,15 @@
 
     private final ItemSetting<Boolean> useEnchantLevelLimit = new ItemSetting<>(this, "use-enchant-level-limit", false);
     private final IntRangeSetting enchantLevelLimit = new IntRangeSetting(this, "enchant-level-limit", 0, 10, Short.MAX_VALUE);
+    private final ItemSetting<List<String>> cantDisenchantLores = new ItemSetting<>(this, "cant-disenchant-lores", asList("§7- §cCan't be Auto-Disenchanted"));
```

Comment:
Don't use `§`. It's a deprecated char and has been for years. It isn't used in any text and hasn't been for a long time.